src/pages/padteambuilder.tsx

```diff
@@ -86,8 +86,8 @@ const PadTeamBuilderPageContainer = React.forwardRef((props, ref) => {
         <FlexRowC gap="2rem">
           <GameConfigSelector />
           <DefaultLevelSelector />
-          <FlexRowC gap="0.25rem">
-            Export:
+          <FlexRowC gap="0.25rem" className={css`cursor:pointer`}>
+          Export: 
```

Comment:
This text doesn't have any event on it, so it should not have `cursor:pointer;` on it. The ticket requests `cursor:pointer;` on the *button*, which begins on the next line.